<launch>
    
    <node name="vive_localization" pkg="vive" type="vive_localization" output="screen">
        <rosparam file="$(find vive)/param/vive_localization.yaml" command="load" />
    </node>


</launch>